tests/executables_src/testScalarRegisterAccessor.cpp

```diff
@@ -135,6 +162,15 @@ BOOST_AUTO_TEST_CASE(testIntRegisterAccessor) {
   accessor.write();
   BOOST_CHECK(dummy == 119);
 
+  // test readAndGet
+  accessor = 470;
+  accessor.write();
+  BOOST_CHECK(accessor.readAndGet() == 470);
+
+  // test setAndWrite
+  accessor.setAndWrite(4711);
```

Comment:
This test is not testing setAndWrite with a version number.  The version number has to be checked for both cases (default argument and given).